qa/pull-tester/rpc-tests.py

```diff
@@ -245,7 +245,8 @@ def option_passed(option_without_dashes):
     'sighashmatch',
     'getlogcategories',
     'getrawtransaction',
-    Disabled('electrum_basics', "Needs to be skipped if electrs is not built")
+    Disabled('electrum_basics', "Needs to be skipped if electrs is not built"),
+    Disabled('electrum_reorg', "Needs to be skipped if electrs is not built")
```

Comment:
Can we add the same method we use for the zmq tests (see ENABLE_ZMQ in rpc-tests.py)...so we would create an ENABLE_ELECTR variable when we build and that would get picked up in the python ./rpc-tests.py when it runs.

Reply:
That's a good idea (for another PR)